src/java/org/apache/cassandra/db/RangeTombstoneList.java

```diff
@@ -58,26 +60,31 @@ public class RangeTombstoneList implements Iterable<RangeTombstone>, IMeasurable
     private ClusteringBound<?>[] starts;
     private ClusteringBound<?>[] ends;
     private long[] markedAts;
-    private int[] delTimes;
+    private int[] delTimesUnsignedIntegers;
 
     private long boundaryHeapSize;
     private int size;
 
-    private RangeTombstoneList(ClusteringComparator comparator, ClusteringBound<?>[] starts, ClusteringBound<?>[] ends, long[] markedAts, int[] delTimes, long boundaryHeapSize, int size)
+    private RangeTombstoneList(ClusteringComparator comparator, ClusteringBound<?>[] starts, ClusteringBound<?>[] ends, long[] markedAts, long[] delTimes, long boundaryHeapSize, int size)
```

Comment:
We can just remove this private constructor, since we only call it from the `RangeTombstoneList(ClusteringComparator, int)` constructor to build the class with an empty array. That also makes `CassandraUInt.fromLong` unneeded.

Reply:
Good catch

Reply:
Great, thanks. We could also break the long line, which has become even longer than it already was:
```java
private RangeTombstoneList(ClusteringComparator comparator,
                           ClusteringBound<?>[] starts,
                           ClusteringBound<?>[] ends,
                           long[] markedAts,
                           int[] delTimesUnsignedIntegers,
                           long boundaryHeapSize,
                           int size)
```

Reply:
Will do